lang/src/org/partiql/lang/eval/visitors/GroupKeyReferencesVisitorTransform.kt

```diff
@@ -101,47 +131,90 @@ public class GroupKeyReferencesVisitorTransform(
                 metas = node.metas
             )
         }
-    }
 
-    private fun getGroupByKeys(node: PartiqlAst.Expr.Select): Map<String, PartiqlAst.GroupKey> {
+    private fun getGroupByKeys(node: PartiqlAst.Expr.Select): LinkedHashMap<String, PartiqlAst.GroupKey> {
```

Comment:
Is this data structure necessary?

Reply:
Removing the use of GroupKeyReferencesVisitorTransform. You'll like the new revision.